lib/plugins.ts

```diff
@@ -79,7 +79,7 @@ export interface ProtractorPlugin {
    *     for the promise to resolve before continuing.  If the promise is
    *     rejected, an error is logged to the console.
    */
-  postResults?: () => q.Promise<any>;
+  postResults?(): void|q.Promise<void>;
```

Comment:
These can actually return any kind of promise.  How about `void|q.IPromise`?  Also, update the comment to have a return type `{Promise=}`.